tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf8 -*-
+import logging
+
+import pytest
+
+from ocs_ci.utility.prometheus import PrometheusAPI
+# from ocs_ci.ocs.ocp import OCP
+# from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.libtest
+def test_workload_rbd(workload_storageutilization_50p_rbd):
+    logger.info(workload_storageutilization_50p_rbd)
+    prometheus = PrometheusAPI()
```

Comment:
This doesn't look finished

Reply:
Yep, I forget this line there. But I could commit some simple example here though.

Reply:
I will push querying code in another pull request.

Reply:
I have prometheus range query method with example in a separate branch here: https://github.com/mbukatov/ocs-ci/commit/9fca30392082ef07fe9b0eab119435e72b58b13c
This will be part of a pull request with some tests.